docs/releases/3.0.txt

```diff
@@ -54,7 +54,12 @@ Minor features
 :mod:`django.contrib.auth`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* Reduced the amount of required boilerplate code for authentication backends
```

Comment:
Introduced `BaseBackend` to ease creation of custom authentication backends. (Or such)

("required boilerplate" seems a bit negative.)